docs/content/docs/build/containers.md

```diff
@@ -3,28 +3,139 @@ title: Containers
 menuPosition: 2
 ---
 
+## Overview
+
+The container is the primary unit of encapsulation in Fluid. It consists of a collection of Loadable Objects and supporting lifecycle APIs. The container is represented as the `FluidContainer` type.
+
+The below docs are broken in to `Creation & loading` and `Interaction & lifecycle`.
+
 ## Creating & loading
 
-What is expected of the developer with respect to creating and deleting containers?
+Creation and loading happens from specific `[service]-client` packages. Each `[service]-client` package contains a common API structure around container interaction. The common API structure is around defining the container schema and getting the `FluidContainer` object.
+
+For the below scenarios we will be using the `client` to represent the `[service]-client` portion. For specific `[service]-client` usage see their corresponding documentation.
+
+### Container schema
+
+The container schema defines the data that the container instance knows about. It includes `initialObjects` that will synchronously loaded and always available. It also includes a definition of types that can be dynamically created. The same schema definition must be provided for both. For more information on `initialObjects` and dynamically cration see [Data modeling](./data-modeling.md)
+
+```typescript
+const schema = {
+    name: "example-container",
+    initialObjects: {
+        layout: SharedDirectory,
+        text: SharedString
+    },
+    dynamicObjectTypes: [ SharedCell, SharedString ],
+}
+```
 
 ### How to create a container
 
+Containers are created off the `client` instance via the `createContainer` function. This function always takes a config that is specific to the service and a schema object that defines the container.
+
+```typescript
+const schema = {
+    name: "example-container",
+    initialObjects: {
+        layout: SharedDirectory,
+        text: SharedString
+    },
+    dynamicObjectTypes: [ SharedCell, SharedString ],
+}
+const { container, containerServices} = await client.createContainer(/*service config*/, schema);
+```
+
 ### How to load a container
 
-### FluidContainer vs containerServices
+To load the container we created in the above section we must provide the service config as well as the exact same schema definition. The same container schema is required on all subsequent loads to ensure the container is loaded correctly.
```

Comment:
Good callout I should add an explicit section. It depends on what you change but the outcome is almost always bad. Basically it's all the problems around schema versioning.

If you change the initialObjects it will just blow up when it's trying to load them. If you add a new there's no creation flow. If you remove on it will try to load it and fail.

If you remove a dynamicType and that type is being used it will blow up when trying to load that type.

If you add a new dynamicType then use it and another user doesn't have that type they will blow up.